db/migrate/20210310223845_create_contacts.rb

```diff
@@ -0,0 +1,10 @@
+class CreateContacts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :contacts do |t|
+      t.string :message, null: false
```

Comment:
I think it is better to set this field as a `text` type. You can get more info on when to choose text or string for an attribute [here](https://rubyinrails.com/2014/03/16/rails-migration-difference-between-text-and-string/).

Reply:
+1